ISISComputingGroup/nicos-core
# Ticket 2164: Added response when a message is incorrect

https://github.com/ISISComputingGroup/IBEX/issues/2164

To test:

* Use the test client to send a bad command, confirm that there is a response from NICOS
* Do the same for bad JSON 

(To help a couple more commands have been added to the test client)
